react-client/src/components.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { configure, mount } from 'enzyme';
+import App from './components/app';
+import Picture from './components/picture';
+
+configure({ adapter: new Adapter() });
+
+describe('App', () => {
+  const app = mount(<App />);
+
+  test('App should be stateful', () => {
+    expect(app.state('pictures')).toBeDefined();
```

Comment:
Do you have a default value here that wouldn't be defined? An empty string or empty array will still be defined so it might be best to have a different check here, but not sure how how've structured your app.